modules/network-observability-packet-drops.adoc

```diff
@@ -22,15 +22,15 @@ CPU and memory usage increases when this feature is enabled.
 .Example `FlowCollector` configuration
 [source, yaml]
 ----
-apiVersion: flows.netobserv.io/v1alpha1
+apiVersion: flows.netobserv.io/v1beta2
 kind: FlowCollector
 metadata:
   name: cluster
 spec:
   namespace: netobserv
-  deploymentModel: DIRECT
+  deploymentModel: Direct
   agent:
-    type: EBPF
+    type: Ebpf
```

Comment:
@jpinsonneau I just want to check, is this `Ebpf` or `eBPF`? 👀 https://github.com/netobserv/network-observability-operator/pull/511/files#diff-075e1f2f99419c9e546c4d913bf0ae45a12261f069cb18fcb0046c6184558449R9139

Reply:
That should be `eBPF` indeed ! Mixed capitalization should be preserved for names:
https://github.com/netobserv/network-observability-operator/pull/511/files#diff-6144d5c1ab9988d6897de59fe0ad2b6790dd9d0178b8a3b8bca03c628b4fdcf2

https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#constants

I've updated the sample which was incorrect:
https://issues.redhat.com/browse/NETOBSERV-1374
https://github.com/netobserv/network-observability-operator/pull/518

Thanks for pointing that @skrthomas !

Reply:
```suggestion
    type: eBPF
```

Reply:
Thanks @jpinsonneau ! I'm glad I can help :)